simulator/docs/env-variables.md

```diff
@@ -6,36 +6,6 @@ used to configure kube-scheduler-simulator.
 Please refer [docker-compose.yml](./../../docker-compose.yml) as an example
 use.
 
-`PORT`: (required) This is the port number on which kube-scheduler-simulator
```

Comment:
I don't think we should remove them soon. Just marking as deprecated is enough for now. We can remove it later.